Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,4381 @@
+import urllib3
+from abc import ABCMeta
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from typing import Callable, Tuple
+from collections import defaultdict
+from bs4 import BeautifulSoup
+from datetime import timezone
+import hashlib
+urllib3.disable_warnings()
+# Note: time.time_ns() is used instead of time.time() to avoid the precision loss caused by the float type.
+# Source: https://docs.python.org/3/library/time.html#time.time_ns
+
+""" CONSTANTS """
+JIRA_INCIDENT_TYPE_NAME = 'Jira Incident'
+ISSUE_INCIDENT_FIELDS = {'issue_id': 'The ID of the issue to edit',
+                         'summary': 'The summary of the issue.',
+                         'description': 'The description of the issue.',
+                         'labels': 'A CSV list of labels.',
+                         'components': 'A CSV list of components',
+                         'priority': 'A priority name, for example "High" or "Medium".',
+                         'due_date': 'The due date for the issue (in the format 2018-03-11).',
+                         'assignee': 'The name of the assignee. Relevant for Jira Server only',
+                         'status': 'The name of the status.',
+                         'assignee_id': 'The account ID of the assignee. Use'
+                                        ' the jira-get-id-by-attribute command to get the user\'s Account ID.',
+                         'original_estimate': 'The original estimate of the Jira issue.'
+                         }
+DEFAULT_FETCH_LIMIT = 50
+DEFAULT_FIRST_FETCH_INTERVAL = '3 days'
+DEFAULT_FETCH_INTERVAL = 1  # Unit is in minutes
+DEFAULT_PAGE = 0
+DEFAULT_PAGE_SIZE = 50
+# Errors
+OAUTH2_AUTH_NOT_APPLICABLE_ERROR = ('This command is not applicable to run with a Jira On Prem instance since On Prem instances'
+                                    ' do not use OAuth2 for authorization.')
```

Comment:
```suggestion
OAUTH2_AUTH_NOT_APPLICABLE_ERROR = ('This command is not applicable to run with a Jira on-prem instance since on-prem instances'
                                    ' do not use OAuth2 for authorization.')
```

Reply:
We actually don't need this error message, since on-prem does indeed use OAuth2.0